db/migrate/20171221143530_create_match_results.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMatchResults < ActiveRecord::Migration[5.1]
+  def change
+    create_table :match_results do |t|
+      t.integer :match_id
+      t.boolean :winner_advances
+
+      t.timestamps
```

Comment:
Since there appears to be no way to change the timestamps of a matchresult, do we really need those, or can we just rely on the timestamps of the match?